src/neptune/handler.py

```diff
@@ -574,36 +570,35 @@ def fetch_values(self, *, include_timestamp: Optional[bool] = True):
         Returns:
             ``Pandas.DataFrame``: containing all the values and their indexes stored in the series field.
 
-        .. _Field types docs page:
+        For more information on field types, see the docs:
            https://docs.neptune.ai/api-reference/field-types
         """
         return self._pass_call_to_attr(function_name="fetch_values", include_timestamp=include_timestamp)
 
     @check_protected_paths
     def delete_files(self, paths: Union[str, Iterable[str]], *, wait: bool = False) -> None:
-        """Delete the file or files specified by paths from the `FileSet` stored on the Neptune servers.
+        """Deletes the files specified by the paths from the `FileSet` stored on the Neptune servers.
 
         Args:
             paths (str or collection of str): `Path` or paths to files or folders to be deleted.
-                Note that these are paths relative to the FileSet itself e.g. if the `FileSet` contains
-                file `example.txt`, `varia/notes.txt`, `varia/data.csv` to delete whole subfolder you would pass
+                Note that the paths are relative to the FileSet itself e.g. if the `FileSet` contains
+                file `example.txt`, `varia/notes.txt`, `varia/data.csv`, to delete the entire subfolder, you would pass
```

Comment:
We can do better here.
```suggestion
                Note that the paths are relative to the FileSet itself. For example, if the `FileSet` contains the
                files `example.txt`, `varia/notes.txt`, `varia/data.csv`, to delete the entire varia subfolder, you would pass
```